app/controllers/application_controller.rb

```diff
@@ -208,4 +193,23 @@ def formats_with_html_fallback
     request.formats.map(&:ref) + [:html]
   end
 
+  def render_403(_exception)
+    byebug
```

Comment:
I don't think this should be here - do we have a spec that would catch something like this?